datastore/google/cloud/datastore/_http.py

```diff
@@ -61,6 +62,9 @@
 
 _DISABLE_GRPC = os.getenv(DISABLE_GRPC, False)
 _USE_GRPC = _HAVE_GRPC and not _DISABLE_GRPC
+_DATASTORE_DIST = get_distribution('google-cloud-datastore')
+_CLIENT_INFO = connection_module.CLIENT_INFO_TEMPLATE.format(
+    _DATASTORE_DIST.version)
```

Comment:
The bare version number seems odd:  I guess the fact that `gl-python` is there plus the fact that we are talking to the Datastore API is enough information?